packages/core/database/lib/entity-manager/regular-relations.js

```diff
@@ -292,14 +293,15 @@ const cleanOrderColumnsForOldDatabases = async ({
   const { joinColumn, inverseJoinColumn, orderColumnName, inverseOrderColumnName } = joinTable;
 
   const now = new Date().valueOf();
+  const randomHex = randomBytes(16).toString('hex');
 
   if (hasOrderColumn(attribute) && id) {
-    const tempOrderTableName = `tempOrderTableName_${now}`;
+    const tempOrderTableName = `orderTable_${now}_${randomHex}`;
     try {
       await db.connection
         .raw(
           `
-          CREATE TEMPORARY TABLE :tempOrderTableName:
+          CREATE TABLE :tempOrderTableName:
```

Comment:
Just wondering, how come we no longer use Temporary Table here but we do below? L341